libs/util/app-config/src/lib/parse-utils.ts

```diff
@@ -121,3 +123,20 @@ export function parseMultiConfigSection(
   }
   return result
 }
+
+export function checkMetadataLanguage(
+  parsedConfigSection: any,
+  outErrors: string[]
+) {
+  if (
+    LANGUAGE_CODES_ISO_3.indexOf(
+      parsedConfigSection.metadata_language.toLowerCase()
+    ) === -1
+  ) {
+    outErrors.push(
+      'In the [global] section: metadata_language is not in ISO 639-2/B format'
```

Comment:
Maybe added the actual value in the error message, to make this even more explicit?

Reply:
Ok, added the value to the message and used the warnings instead of errors, since this is rather an "unexpected" value than a "mandatory value missing".